.github/workflows/build_iso.yml

```diff
@@ -0,0 +1,62 @@
+name: Build Cosmic ISOs
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/build_iso.yml'
+
+env:
+  IMAGE_REGISTRY: ghcr.io/${{ github.repository_owner }}
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build-iso:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+    container:
+      image: fedora:${{ matrix.major_version }}
+      options: "--privileged"
+      volumes:
+        - "/:/host"
+    strategy:
+      fail-fast: false
+      matrix:
+        image_name: [cosmic-silverblue, cosmic-base]
+        image_tag: [40-amd64, rawhide-amd64]
+        include:
+          - image_tag: 40-amd64
+            major_version: 40
+          - image_tag: rawhide-amd64
+            major_version: 41 # We use F41 for rawhide
```

Comment:
@noelmiller Do you know what version we need to put for rawhide builds?  Currently getting the following when using 41

```
2024-03-03 00:04:04,463: Using platform:f41 for module_platform_id
2024-03-03 00:04:04,465: No enabled repos
error: unable to create the dnf base object
```

Reply:
Try rawhide

Reply:
I was worrying `rawhide` would cause issues in the `lorax -V` flag as I believe it only works with ints, but hopefully I'm wrong and it's easy enough to enable

Reply:
Strange...  Using `rawhide` must alias to 41

```
lorax -p cosmic-base -v rawhide -r rawhide -t Silverblue \
          --isfinal --buildarch=x86_64 --volid=cosmic-x86_64-rawhide-amd64 \
          --nomacboot \
          --repo /etc/yum.repos.d/fedora.repo \
          --repo /etc/yum.repos.d/fedora-updates.repo \
          --add-template /__w/_actions/ublue-os/isogenerator/1.0.9/lorax_templates/set_installer.tmpl \
	  --add-template /__w/_actions/ublue-os/isogenerator/1.0.9/lorax_templates/configure_upgrades.tmpl \
	  --add-template /__w/_actions/ublue-os/isogenerator/1.0.9/lorax_templates/secure_boot_key.tmpl \
          /__w/_actions/ublue-os/isogenerator/1.0.9/results/
2024-03-03 00:15:42,190: selinux is Disabled
2024-03-03 00:15:42,191: Using platform:f41 for module_platform_id
```

Reply:
also, thing to remember, version and release are for the installer ISO that is being built. You can install something that is Fedora 40 using a Fedora 39 ISO since we are just deploying the container OCI dir to the filesystem.

Reply:
:eyes:  I didn't consider that!  
Will use 40 for the major_version of rawhide

Reply:
Also, I have no clue whether installing to an ARM device would actually work :D

Reply:
I'm leaving that for another day as I'm unable to test it.
I doubt it will work without some hackery in the scripts.